gnome-extra/gnome-commander/gnome-commander-1.10.0.ebuild

```diff
@@ -52,6 +51,15 @@ src_configure() {
 		$(use_with unique)
 }
 
+pkg_setup() {
+        if tc-is-gcc; then
```

Comment:
I think it can be made into one `if`
`if tc-is-gcc && [[ $(gcc-major-version) -lt 8 ]] ; then`

Reply:
Right, I changed that.